apps/admin-gui/src/app/vos/pages/member-detail-page/member-overview/member-overview.component.html

```diff
@@ -105,9 +106,17 @@ <h6 class="d-flex">
               <tr *matHeaderRowDef="['sponsors', 'expiration']" mat-header-row></tr>
               <tr *matRowDef="let sponsor; columns: ['sponsors', 'expiration'];" mat-row></tr>
             </table>
-          </mat-card-content>
-        </mat-card>
-      </div>
+          </div>
+          <button
```

Comment:
Could you please add a tooltip when the button is disabled (similar to how it is on the sponsored members page). Information whether the user is not a sponsor/no sponsors are available should be easily retrievable in the `isSponsorButtonEnabled` method.